notebooks/text_models/solutions/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>* "It can be" --&gt; "it can be"</p><p>* Not sure I understand what this sentence means: "Since this text preprocessor is a TensorFlow model, It can be included in your model directly."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Re-phrased the sentence to make it clearer.</p>